README.md

```diff
@@ -110,32 +110,33 @@ This repo relies on the following tools being installed:
 
 ### Making everything
 
-Run the following command to setup the project:
+Run the following command to set up the project:
 
 ```shell
 npm run clean
 npm install
 npm run setup
+./make.sh
```

Comment:
I guess this fails if no shell is installed on Windows? (for me it started git bash).